dir.props

```diff
@@ -229,7 +220,17 @@
     <WindowsAppContainer>false</WindowsAppContainer>
 
     <DefineConstants>$(DefineConstants);RUNTIME_TYPE_NETCORE</DefineConstants>
+    
+    <!-- target framework for msbuild nuget packages. Used by Nuspec.ReferenceGenerator.targets to update corefx dependencies-->
+    <NuSpecTfm>netstandard$(TargetFrameworkVersion.Substring(1))</NuSpecTfm>
```

Comment:
I don't love this but we store the target framework version as something like "v1.3" but I want to glue it onto a moniker like "netstandard1.3".  So I strip off the first character.  Maybe this should be TrimStart("v") ?